src/test/java/teammates/ui/webapi/FeedbackSessionClosingRemindersActionTest.java

```diff
@@ -103,14 +105,40 @@ public void testExecute() throws Exception {
         session3.setSentOpenEmail(false); // fsLogic will set the flag to true
         verifyPresentInDatabase(session3);
 
+        // update deadline extensions to have end time within the next 24 hours
+        DeadlineExtensionAttributes deadlineExtensionStudent =
+                typicalBundle.deadlineExtensions.get("student4InCourse1Session1");
+        DeadlineExtensionAttributes deadlineExtensionStudentAlreadyAnswered =
+                typicalBundle.deadlineExtensions.get("student3InCourse1Session1");
+        DeadlineExtensionAttributes deadlineExtensionInstructor =
+                typicalBundle.deadlineExtensions.get("instructor2InCourse1Session1");
+        List<DeadlineExtensionAttributes> deadlineExtensions = List.of(
+                deadlineExtensionStudent,
+                deadlineExtensionStudentAlreadyAnswered,
+                deadlineExtensionInstructor);
+
+        for (DeadlineExtensionAttributes deadlineExtension : deadlineExtensions) {
```

Comment:
var or forEach?

Reply:
We should be careful not to overuse var everywhere. There isn't really much value to replacing explicit types with var in such cases, especially if the line is fairly short. In addition, var is not used consistently throughout the project. In this case, explicit types still provide extra contextual information. Is this `DeadlineExtension` or `DeadlineExtensionAttributes`?

Reply:
Im only raising this because the type is literally at line 115. This kind is ok to me, and we should start using Java 11 features?

Reply:
It is communicated with Wilson that this can be used discreetly. I do believe having a type here when it is clear that `deadlineExtensions` is a `List<DeadlineExtensions>` is redundant.